### DISCUSSION TITLE
register

### DISCUSSION THREAD

**Discussion by Joop-bit** [iterate-ch/cyberduck#16133] [Q&A]
I cannot register. How to continue?
joop.jansen05@gmail.com

**Comment by tatze96** [comment]
To register Cyberduck to you, download the registration key file from the email attachments and double-click on it. Ensure that the registration key file wasn't changed by your email application and has the file extension _.cyberducklicense_. If the issue persists, send us an email to support@cyberduck.io for further investigation.
